Mediator/Main.java

```diff
@@ -0,0 +1,20 @@
+package com.clearfaun.mdlag;
+
+public class Main {
+
+    public static void main(String[] args) {
+	// write your code here
+
+
```

Comment:
Overall comment: This time you did a poor job in the implementation. Your implementation looks like a mediator pattern but is not because you are missing some parts: like the communication between concrete colleague through mediator. It would be nice you create an interface for the concrete colleague so that you follow the pattern intention: decoupling, encapsulation and flexibility.